src/middlewared/middlewared/plugins/account.py

```diff
@@ -287,6 +289,84 @@ async def query(self, filters, options):
             filter_list, result, filters, options
         )
 
+    @private
+    def validate_homedir_path(self, verrors, schema, data, users):
+        needs_additional_validation = True
+        p = Path(data['home'])
+
+        if not p.is_absolute():
+            verrors.add(f'{schema}.home', '"Home Directory" must be an absolute path.')
+            return False
+
+        if p.is_file():
```

Comment:
`p.is_file()` will return False even if the file/dir doesn't exist so there is a potential the validation error that is returned to end-user leads them astray. Probably want to do `if not p.exists()` first so by the time `p.is_file()` is called, ideally, it exists 😄

Reply:
Except we want to error-out if `is_file()` is True (which only happens if the path exists and is a file -- and not a directory).